legacy/src/scss/_patterns_cores-chart.scss

```diff
@@ -7,6 +7,7 @@ $bar-height: 11px;
 @mixin maas-p-cores-chart {
   maas-cores-chart {
```

Comment:
Huh is this used? Neither a placeholder or class?

Reply:
It's weird, but yeah this is used in the code. It's kinda like css specific for Angular's version of web components.